tests/test_cli.py

```diff
@@ -218,6 +219,51 @@ def test_cli_input_errors(capsys, expected_help, cmd, expect_msg, should_show_he
     assert err.rstrip().endswith(expect_msg)
 
 
+# These commands use the new syntax with the legacy commands
+@pytest.mark.parametrize(
+    "cmd",
+    (
+        "install linux desktop 5.10.0",
+        "install linux desktop 5.10.0 gcc_64",
+        "src linux desktop 5.10.0",
+        "doc linux desktop 5.10.0",
+        "example linux desktop 5.10.0",
+        "tool windows desktop tools_ifw",
+        "tool windows desktop tools_ifw qt.tools.ifw.31",
```

Comment:
This is the wrong way to test `aqt tool`; I'm not sure how to fix it yet. This test relies on `argparse` to raise `SystemExit` when the arguments are invalid. `aqt tool` is supposed to take 4 positional arguments and `argparse` does not validate the last 3. 

If you run `aqt tool windows desktop tools_ifw qt.tools.ifw.31`, the correct behavior is for `aqt tool` to try to fetch a tool called `desktop` with version `tools_ifw` and variant `qt.tools.ifw.31`.